src/App.js

```diff
@@ -10,20 +10,32 @@ import Scorecard from './Components/Scorecard'
 import Courses from './Components/Courses'
 
 const App = () => {
+  const [ discContext, setDiscContext ] = useState({
+    name: 'Discify',
```

Comment:
I believe this defines the initial state of app. I wonder how much we need the context in it's own file?